packages/acs-ui-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internal/acs-ui-common",
-  "version": "1.2.2-beta.1",
+  "version": "1.3.0",
```

Comment:
These all need bumped to the next beta.0 version

Reply:
isnt that supposed to happen when the next release is kicked off?